src/panels/config/voice-assistants/ha-config-voice-assistants-expose.ts

```diff
@@ -199,7 +199,7 @@ export class VoiceAssistantsExpose extends SubscribeMixin(LitElement) {
         template: () =>
           html`<ha-icon-button
             @click=${this._removeEntity}
-            .path=${mdiMinusCircleOutline}
+            .path=${mdiCloseCircleOutline}
```

Comment:
shouldnt this be box then too?

Reply:
Not sure
![CleanShot 2023-04-26 at 16 52 20](https://user-images.githubusercontent.com/5878303/234615032-0910e389-b22d-41df-a8c8-6b5bddaa1af6.png)